node/lib/deltachat.ts

```diff
@@ -19,10 +19,11 @@ interface NativeAccount {}
 export class AccountManager extends EventEmitter {
   dcn_accounts: NativeAccount
   accountDir: string
+  json_rpc_started = false
```

Comment:
`jsonRpcStarted`. Let's stick to camelCase for everything in JS land apart from direct references to FFI names.